static/elements/chromedash-approvals-dialog.js

```diff
@@ -171,6 +171,15 @@ class ChromedashApprovalsDialog extends LitElement {
     return dateStr.split('.')[0]; // Ignore microseconds.
   }
 
+  findStateName(state) {
+    for (let item of STATE_NAMES) {
+      if (item[0] == state) {
+        return item[1];
+      }
+    }
+    throw new Error(`Unexpected approval state ${state}`);
```

Comment:
Do we need to do anything here, like an alert?

(if the render part of the code throws error, will it just result in a blank/noninteractive component?)

Reply:
I made this fail in a less dramatic way by just displaying the state number to the user.  In practice that should never happen, but in the case of a defect or some kind of data corruption, the dialog will still load and we will be able to see visible evidence of the problem.